dockerfiles/alpine/Dockerfile

```diff
@@ -39,23 +48,25 @@ RUN echo "fn main() {println!(\"if you see this, the build broke\")}" > bob/src/
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/brt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-apps/bin/bobt.rs \
   && echo "fn main() {println!(\"if you see this, the build broke\")}" > bob-access/src/lib.rs \
-  && cargo build --release --target=x86_64-unknown-linux-musl
+  && cargo build --profile=${BUILD_PROFILE} --target=x86_64-unknown-linux-musl
 
 
 # separate stage for proto build
 RUN echo "fn main() {println!(\"if you see this, the build broke\")} pub mod grpc {include!(\"bob_storage.rs\");}" > bob-grpc/src/lib.rs \
   && mkdir -p bob-grpc/proto
+RUN rm -f bob-grpc/src/bob_storage.rs
```

Comment:
I just read the dockerfile and didn't find the place where this file gets copied before this line. Actual problem is that the file overrides by the `COPY . .` instruction below. The proper way to fix is to add `bob-grpc/src/bob_storage.rs` into `.dockerignore` file in the repository root